CHANGELOG.md

```diff
@@ -25,6 +25,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 - Added support for compiling `jsx` and `tsx` modules in the playground.
 
+### Changed
+
+- Updated the tsconfig in `language-service-context.ts` to include `allowJs` and `jsx`.
```

Comment:
This file is to be consumed by users, not so much us, so I think this is too much detail. The user only cares that tsx and jsx files are now compiled.